### DISCUSSION TITLE
Add other information in value and metadata field

### DISCUSSION THREAD

**Discussion by manishlok** [maize-tech/laravel-markable#6] [Q&A] [answered]
Very handy plugin !
I have a question. How to Add other information in like value and metadata field. As migration and table holds this, how to implement this fields?

**Comment by riccardodallavia** [comment, marked as answer]
Hi @manishlok !

You can use the mark values as explained in the docs [here](https://github.com/maize-tech/laravel-markable#working-with-mark-values).
We are also working on the functionality to add metadata to marks!
